packages/veraswap-app/src/atoms/orbiter.ts

```diff
@@ -0,0 +1,102 @@
+import { atom } from "jotai";
+import { chainIdToOrbiterChainId, OrbiterParams, orbiterRoutersQueryOptions } from "@owlprotocol/veraswap-sdk";
+import { atomWithQuery } from "jotai-tanstack-query";
+import { zeroAddress, Address, parseUnits } from "viem";
+import { tokenInAmountAtom, tokenInAtom, tokenOutAtom } from "./tokens.js";
+
+export const orbiterRoutersAllAtom = atom((get) => {
+    const orbiterRoutersMainnet = atomWithQuery(() => orbiterRoutersQueryOptions(true));
+    const orbiterRoutersTestnet = atomWithQuery(() => orbiterRoutersQueryOptions(false));
+
+    return [...(get(orbiterRoutersMainnet).data ?? []), ...(get(orbiterRoutersTestnet).data ?? [])];
+});
+
+export const orbiterRoutersEndpointContractsAtom = atom((get) => {
+    const orbiterRoutersAll = get(orbiterRoutersAllAtom);
+
+    return orbiterRoutersAll
+        .filter((router) => router.srcToken === zeroAddress && !!router.endpointContract)
+        .reduce((acc, curr) => {
+            return acc.set(Number(curr.srcChain), curr.endpointContract as Address);
```

Comment:
This seems `Map<number, Address>` seems not best suited:
* `Map` isn't necessary, a simple `Record<number, Address>` suffices and benefits from ease of use with things like spread operator, mutation if needed etc...
* Filters out routers for non-native tokens (eg. USDC)
* Assumes 1 router per source chain
* Loose a bunch of other info (available, filler address etc...)

I think this atom can be omitted as can be seen by `orbiterRouterAtom` not using this.

Reply:
there is 1 router per source chain
Needed to watch for potential transfer events, see https://sepolia.etherscan.io/tx/0xec5e7fd7921176f071ff2fde6828221c14f25c6ef1eacdad26f419e133b94db2#61